assets/scss/_code.scss

```diff
@@ -25,6 +25,7 @@
         padding: 0.2em 0.4em;
         margin: 0;
         font-size: 85%;
+	word-break: normal;
```

Comment:
for this to resolve the actual issue in #75, the `word-break: normal;` must also applied to `tr` (and not specific to a child of `p`) for example:

Include `tr code` (without the `>` child selector):
```
// Inline code
    p code, li > code, td > code, th > code, tr code {
...
```

Or alternatively, just apply it to all elements of a table  (use `table code`):
```
// Inline code
    p code, li > code, table code {
...

```